### DISCUSSION TITLE
Hide Address Bar setting

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferdium/ferdium-app#1700] [Ideas]
Please add a setting to hide the address bar but to keep the back button, forward button, reload button, open in browser button and home button.

**Comment by vraravam** [comment]
This is already achievable via the Preferences UI
<img width="653" alt="Screenshot 2024-04-17 at 9 04 59 AM" src="https://github.com/ferdium/ferdium-app/assets/69629/698474a2-803b-4eb8-8fe2-21412fccb5f4">

**Reply by trymeouteh** [reply]
I would like the navigation bar to always be visible, just not the address bar inside the navigation bar

**Comment by SpecialAro** [comment]
I'm also closing this discussion. Please refrain from adding duplicate topics in several places

https://github.com/ferdium/ferdium-app/issues/1708
